service/controller/resource/s3object/desired.go

```diff
@@ -23,7 +34,55 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	var clusterCerts certs.Cluster
+	var images v_4_9_1.Images
+	{
+		releaseVersion := customObject.Labels[label.ReleaseVersion]
+		releaseName := fmt.Sprintf("v%s", releaseVersion)
+		release, err := r.g8sClient.ReleaseV1alpha1().Releases().Get(releaseName, metav1.GetOptions{})
```

Comment:
It's not totally clear where the boundary between the operator and the k8scloudconfig logic should be. We don't want to repeat the release->images code in every operator so some of this should probably be moved to k8scc.

Reply:
I'm happier with the split here now.